### PR TITLE
Fix false failures in ItIntrospectVersion tests due to k8s delay in updating the metadata

### PR DESCRIPTION
**Pull request by sankarpn**
The testUpdateImageName test in ItIntrospectVersion class has 2 clusters and when the test updates the image used by the domain resource it rolls all of the pods in the domain. But the test only waits for one of the cluster pods to roll and it causes failures in subsequent test when the subsequent test checks whether the second cluster has rolled.

The fix is to wait for all pods in the domain to roll and conclude the tests.

https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/9012

### CODE REVIEW

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java [hunk 1]
```diff
@@ -1039,11 +1067,11 @@ void testDedicatedModeSameNamespaceScale() {
   /**
    * Update the introspectVersion of the domain resource using lifecycle script.
    * Refer to kubernetes/samples/scripts/domain-lifecycle/introspectDomain.sh
-   * The usecase update the introspectVersion by passing differnt value to -i 
-   * option (non-numeic, non-numeric with space, no value) and make sure that 
-   * the introspectVersion is updated accrodingly in both domain sepc level 
-   * and server pod level. 
-   * It also verifies the intospector job is started/stoped and none of the 
+   * The usecase update the introspectVersion by passing differnt value to -i
+   * option (non-numeic, non-numeric with space, no value) and make sure that
```
Comment (anpanigr) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java:1071]: non-numeic -> non-numeric

**Review by anpanigr — COMMENTED** [review]
Start a complete suite run.

**Review by vanajamukkara — APPROVED** [review]
LG

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java (hunk 1 above)
Comment (sankarpn) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java]: fixed typos.

**Review by anpanigr — APPROVED** [review]

**Pull request merged**